common-service-core/src/main/java/org/zowe/apiml/constants/ApimlConstants.java

```diff
@@ -22,8 +22,6 @@ private ApimlConstants() {
     public static final String BASIC_AUTHENTICATION_PREFIX = "Basic";
     public static final String BEARER_AUTHENTICATION_PREFIX = "Bearer";
     public static final String PAT_COOKIE_AUTH_NAME = "personalAccessToken";
```

Comment:
Didn't we want to remove also special cookie name for the PAT?

Reply:
Yes, we want to remove it. But right now it would be braking change. I think it will be an item for v3.